RockPaperScissors/RockPaperScissors.cs

```diff
@@ -14,12 +14,62 @@ public static void Main()
 
             // leave this command at the end so your program does not close automatically
             Console.ReadLine();
+            
+        
         }
         
         public static string CompareHands(string hand1, string hand2)
         {
             // Your code here
+            if (hand1 == "rock" && hand2 == "rock")
+            {
+                Console.WriteLine("Tie");
+            } else if (hand1 == "rock" && hand2 == "scissors")
+            {
+                Console.WriteLine("hand1 wins!");
+            } else if (hand1 == "rock" && hand2 == "paper")
+            {
+                Console.WriteLine("hand2 wins!");
+            } else if (hand1 == "scissors" && hand2 == "rock")
+            {
+                Console.WriteLine("hand2 wins");
+            } else if (hand1 == "scissors" && hand2 == "paper")
+            {
+                Console.WriteLine("hand1 wins");
+            } else if (hand1 == "scissors" && hand2 == "scissors")
+             {
+                 Console.WriteLine("Tie");
+             } else if (hand1 == "paper" && hand2 == "paper")
+             {
+                 Console.WriteLine("Tie");
+             } else if (hand1 == "paper" && hand2 == "scissors")
+             {
+                 Console.WriteLine("hand2 wins");
+             } else if (hand1 == "paper" && hand2 == "rock")
+             {
+                 Console.WriteLine("hand1 wins");
+             } else
+             {
+                 Console.WriteLine("try again");
+             }
+        
+            
             return hand1 + ' ' + hand2;
         }
+        // public static bool test()
```

Comment:
I see the tests defined, but the way you implemented CompareHands, these tests would not work.